core/storage/btree.rs

```diff
@@ -365,6 +379,10 @@ pub struct BTreeCursor {
     reusable_immutable_record: RefCell<Option<ImmutableRecord>>,
     empty_record: Cell<bool>,
     pub index_key_sort_order: IndexKeySortOrder,
+    /// Stores the last record that was seen.
```

Comment:
This isn't a correct comment, right? We should explain that it is stored when rebalancing happens